pkg/scalers/selenium_grid_scaler.go

```diff
@@ -65,10 +66,12 @@ type seleniumSession struct {
 type capability struct {
 	BrowserName    string `json:"browserName"`
 	BrowserVersion string `json:"browserVersion"`
+	PlatformName   string `json:"platformName"`
 }
 
 const (
 	DefaultBrowserVersion string = "latest"
+	DefaultPlatformName   string = "linux"
```

Comment:
I am curious, what is default behaviour before introducing this change? Do we need to enforce linux as the default one? What was the result of the original query if I don't specify linux platform there?

Reply:
Defining `linux` as the default platform name instead of '' is needed when there is a queued session in selenium hub whose capability platformName is `Linux`:
```json
{"browserName":"chrome","browserVersion":"105.0","platformName":"LINUX"}
```
In this case, if the platformName does not have a default value defined as 'linux', the match between the queued session and the ScaledObject will never occur.

Reply:
And is there a way when the queued session doesnt have capability specified? what would happen then?

Reply:
The behavior will be the same as before this implementation, that is, the scaledObjects filter will only consider browserName and browserVersion (if defined)

Reply:
Please disregard, I notice that you evaluate that in the code :)